core/src/main/java/com/sequenceiq/cloudbreak/controller/validation/template/TemplateValidator.java

```diff
@@ -187,6 +191,14 @@ private boolean isComputeInstanceGroup(InstanceGroup instanceGroup) {
         return GROUP_NAME_COMPUTE.equalsIgnoreCase(instanceGroup.getGroupName());
     }
 
+    private boolean isCoordinatorInstanceGroup(InstanceGroup instanceGroup) {
+        return GROUP_NAME_COORDINATOR.equalsIgnoreCase(instanceGroup.getGroupName());
```

Comment:
we should not ever match groupnames. Customers can create their own blueprint and this is hidden magic. Please use service name check or rolename.